.travis.yml

```diff
@@ -31,6 +34,9 @@ matrix:
       env: WP_TRAVISCI=phpcs
     - php: 5.3
       env: WP_VERSION=latest
+  allow_failures:
+    - php: 5.6
+      env: WP_TRAVISCI=phpcs
```

Comment:
If you want to allow failures for `phpcs`, you can do so by modifying your `.travis.yml` after scaffolding. I'd like to keep failures as the default behavior, to encourage following WPCS